frontend/lib/src/components/core/Block/Block.tsx

```diff
@@ -325,6 +328,10 @@ const VerticalBlock = (props: BlockPropsWithoutWidth): ReactElement => {
     ...props,
     ...{ width },
   }
+  // Extract the user-specified key from the block ID (if provided):
+  console.log("props.node.deltaBlock.id", props.node.deltaBlock.id)
```

Comment:
left-over `console.log`

Reply:
ups, we should add some linting rule for that